data/projects.js

```diff
@@ -1,10 +1,38 @@
 projects = [
 	{
-		"name" : "Project 1",
-		"desc" : "Desc",
-		"mentors" : [
-		"user1",
-		"user2"
+		"name" : "Improve Diff Handling",
+		"desc" : "Bears should be able to offer more than just one possible patch \
+for an issue. Users should be able to choose between different patches, e.g. \
+for different possible spelling corrections.\
+\
+In addition bears should be able to offer templated patches, e.g. to offer \
+variable renamings while the user should be asked for the new variable name.",
```

Comment:
With all these linebreaks the code is not nice to read. Any better formatting options?